flask/helpers.py

```diff
@@ -958,3 +958,33 @@ def total_seconds(td):
     :rtype: int
     """
     return td.days * 60 * 60 * 24 + td.seconds
+
+def is_IP(string, var_name):
```

Comment:
PEP8: should be `is_ip`

Also, the naming is very different from what the function does.